candidates/kusama.json

```diff
@@ -5885,6 +5885,27 @@
       "stash": "FvQau5sffdGLfDhzmjLajgSMDURVjDyDMG752BTyXr69Tdv",
       "riotHandle": "@skalablex:matrix.org",
       "kyc": true
+    },
+    {
+      "slotId": 845,
+      "name": "snail",
+      "stash": "DGDpqWqFwQRkgPVc5JxBZ4ZyyGqcTQ7PNxLU6Kyi84RP5H4",
+      "riotHandle": "@snail773:matrix.org",
+      "kyc": true
+    },
+    {
+      "slotId": 846,
+      "name": "snail-2",
+      "stash": "HWb4f3AXUomEQN7xBeAc6thRjgxKKgmFFdM3ZCwCsWRdo5w",
+      "riotHandle": "@snail773:matrix.org",
+      "kyc": true
+    },
+    {
+      "slotId": 847,
```

Comment:
@kubaw3f Good Karma is [already present](https://github.com/w3f/1k-validators-be/blob/master/candidates/kusama.json#L817), was that a mistake ?